disco_aws_automation/disco_alarm_config.py

```diff
@@ -117,7 +117,21 @@ def decode_alarm_name(name):
         Raises AlarmConfigError if unable to parse the alarm name.
         """
         parts = name.split("_")
-        if len(parts) == 5:
+        if len(parts) > 5:
+            # alarm names that have extra underscores
+            # Assume this means the metric name portion has underscores
+            # FIXME this won't work if the alarm is missing a team name AND the metric name has underscores
```

Comment:
Missing a team name part would fail even for the condition `len(parts) == 5`. So the assumption is that all new metric will have a team name.